external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -539,3 +623,92 @@ class TosaPartitionPass : public TosaPartitionBase<TosaPartitionPass> {
 std::unique_ptr<Pass> mlir::tosa::createTosaPartitionPass() {
   return std::make_unique<TosaPartitionPass>();
 }
+
+std::unique_ptr<Pass>
+mlir::tosa::createTosaPartitionPass(mlir::tosa::PartitionConfig *config) {
```

Comment:
New create-function with the configuration parameter.

Reply:
If it is required, you can pass as a const reference.